.github/workflows/post-merge-build-and-push.yaml

```
@@ -0,0 +1,45 @@
+name: Build and Push rhdh-e2e-runner Image
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '.ibm/images/**'
+
+concurrency:
+  group: post-merge-build
+  cancel-in-progress: false
+
+env:
+  REGISTRY: quay.io
+  REGISTRY_IMAGE: rhdh-community/rhdh-e2e-runner
+
+jobs:
+  build-and-push:
+    name: Build and Push Image
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Log in to Quay.io
+        uses: docker/login-action@v3
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Build and Push Image
+        uses: docker/build-push-action@v5
+        with:
+          context: .ibm/images/
+          push: true
+          tags: ${{ env.REGISTRY }}/${{ env.REGISTRY_IMAGE }}:latest
```

```suggestion
          tags: ${{ env.REGISTRY }}/${{ env.REGISTRY_IMAGE }}:next
```
I still think we should align the naming with the nightly builds of RHDH.

---

```suggestion
          tags: ${{ env.REGISTRY }}/${{ env.REGISTRY_IMAGE }}:${{ github.ref_name }}
```
tag based on branch name

---

@zdrapela I can handle it on a specific PR